src/game/Language.h

```diff
@@ -299,6 +299,7 @@ enum MangosStrings
 
     LANG_COMMAND_MODIFY_REP             = 305,
     //LANG_COMMAND_MODIFY_ARENA           = 306,
+    LANG_ACCOUNT_WARNED_LOGIN = 306,
```

Comment:
It would be nice if you would format the code accordingly to the surrounding code